package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internetarchive/bookreader",
-  "version": "4.15.0",
+  "version": "4.15.1-alpha.2b5e76c",
```

Comment:
This needs to be reverted. Canary versions can be published for testing, but should not be part of the commit history. The version will be updated once a new release is published.